src/Scenario/PresenceMonitoringScenario/PresenceMonitoringCallbacks.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+
+namespace TelegramOSINT\Scenario\PresenceMonitoringScenario;
+
+
+interface PresenceMonitoringCallbacks
+{
+    /**
+     * @param string $number
+     * @return void
+     */
+    public function onOnline(string $number);
```

Comment:
`: void`, let's be strict typed for new code where possible.

Reply:
++